src/libs/Navigation/types.ts

```diff
@@ -68,7 +68,7 @@ type CentralPaneScreensParamList = {
     // Param types of the search central pane are also used for the search bottom tab screen.
     [SCREENS.SEARCH.CENTRAL_PANE]: {
         q: SearchQueryString;
-        isCustomQuery: boolean;
+        isCustomQuery: string;
```

Comment:
Why this change? `isCustomQuery` implies a bool to me

Reply:
It's a bit misleading because this parameter is stored as a string in `route.params`, and we need to parse it to `boolean` when we read it in the component

Reply:
yeah this looks surprising but I believe its correct to type it like this. Everything that comes from route query params will always be a string I believe